applications/tari_base_node/windows/README.md

```diff
@@ -0,0 +1,60 @@
+# Tari Base Node and Wallet Runtime Instructions
+
+## Pre-requisites 
+
+- SQLite:
+  - Download 32bit/64bit Precompiled Binaries for Windows for [SQL Lite](https://www.sqlite.org/index.html). 
+  - Extract to local path, e.g. `%USERPROFILE%\.sqlite`.
+  - Ensure folder containing `sqlite3.dll`, e.g. `%USERPROFILE%\.sqlite`, is in the path.
+
+- Tor
+  - Donwload [Tor Windows Expert Bundle](https://www.torproject.org/download/tor/).
+  - Extract to local path, e.g. `C:\Program Files (x86)\Tor Services`.
+  - Ensure folder containing the Tor executable, e.g. `C:\Program Files (x86)\Tor Services\Tor`, is in the path.
```

Comment:
Would be a good idea to mention path is in environment variables, also whether it is the environment variable for system or user.